modules/storage/storage-account/blobServices/containers/deploy.bicep

```diff
@@ -0,0 +1,73 @@
+@maxLength(24)
+@description('Required. Name of the Storage Account.')
+param storageAccountName string
+
+@description('Optional. Name of the blob service.')
+param blobServicesName string = 'default'
```

Comment:
It's not. The parameter exists for consistency - with the one default that is allowed. If (in theory) it would ever change, we wouldn't block anything.